### PR TITLE
About editing generated documentation

### PR DESCRIPTION
**Pull request by yecril71pl**
The current statement is false; there is a lot of open code where the documentation can be edited by contributors.

### CODE REVIEW

**Comment by nschonni** [comment]
Non-editable content is not generated, just some of the technical writer teams don't accept external contributions, which is why the current statement is correct

**Comment by yecril71pl** [comment]
> Non-editable content is not generated, just some of the technical writer teams don't accept external contributions, which is why the current statement is correct

API documentation is generated.  In order to improve that, you need to talk to the product team, not to the documentation team. If your contribution makes sense, it will be accepted and translated to on-line documentation using project-specific tools.  That is how it works.

**Comment by nschonni** [comment]
That may be true for some of the dotnet docs APIs, but this site/contribution guide is for all of `docs.microsoft.com`

File: Contribute/index.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Quick edits streamline the process to report and fix small errors and omissions
 
    ![Location of the Edit link](./media/index/edit-article.png)
 
-   If the **Edit** button doesn't appear it means the content is not open to public contributions.
+   If the **Edit** button is not present, it means the content is not open to public contributions.  Some pages are generated, e.g. from inline documentation in code, and must be edited in the project they belong to.
```
Comment (carlyrevier) [inline Contribute/index.md:38]:
```suggestion
   If the **Edit** button isn't present, it means the content isn't open to public contributions.  Some pages are generated, (for example, from inline documentation in code) and must be edited in the project they belong to.

```

File: Contribute/index.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Quick edits streamline the process to report and fix small errors and omissions
 
    ![Location of the Edit link](./media/index/edit-article.png)
 
-   If the **Edit** button doesn't appear it means the content is not open to public contributions.
+   If the **Edit** button isn't present, it means the content isn't open to public contributions.  Some pages are generated, (for example, from inline documentation in code) and must be edited in the project they belong to.
```
Comment (carlyrevier) [inline Contribute/index.md:37]:
```suggestion
   If the **Edit** button isn't present, it means the content isn't open to public contributions.  Some pages are generated (for example, from inline documentation in code) and must be edited in the project they belong to.
```

**Review by carlyrevier — APPROVED** [review]

**Pull request merged**